ambry-server/src/integration-test/java/com.github.ambry.server/ServerTestUtil.java

```diff
@@ -1692,14 +1692,19 @@ private static void checkBlobContent(MockClusterMap clusterMap, BlobId blobId, B
     }
   }
 
-  private static Properties getRouterProps(String routerDatacenter) {
+  /**
+   *
+   * @param routerDatacenter
+   * @return
+   */
+  private static Properties getRouterProps(String routerDatacenter, boolean enableCrossDC) {
```

Comment:
where is `enableCrossDc` false? Also, the javadoc is incomplete.

Reply:
will fix it